languages/de.ini

```diff
@@ -869,7 +866,12 @@ Sensor Image = "Fernerkundungsbilder"
 Serial = "Schriftenreihe"
 Series = "Schriftenreihe"
 Set = "Wechseln"
-Showing = "Treffer"
+showing_items_html = "Treffer <strong>%%start%% - %%end%%</strong> Datensätze"
+showing_items_of_html = "Treffer <strong>%%start%% - %%end%%</strong> von <strong>%%total%%</strong> Datensätze"
+showing_results_for_html = "Treffer <strong>%%start%% - %%end%%</strong> Treffer for search: '<strong>%%lookfor%%</strong>'"
```

Comment:
I think something went wrong with the automatic processing of the German language file -- I'm not sure why "Treffer" is showing up multiple times in some of these sentences. You should probably check whether whatever went wrong here has also affected any of the other translations.